docs/user.md

```diff
@@ -181,3 +181,40 @@ spec:
     operator: Exists
     effect: NoSchedule
 ```
+
+## How to clone an existing PostgreSQL cluster
+
+To spin up a new cluster as a clone of the existing one, you need to provide a
+clone section in the spec:
+
+```yaml
+apiVersion: "acid.zalan.do/v1"
+kind: postgresql
+
+metadata:
+  name: acid-test-cluster
+spec:
+  clone:
+    uid: "efd12e58-5786-11e8-b5a7-06148230260c"
+    cluster: "acid-batman"
+    timestamp: "2017-12-19T12:40:33+01:00"
+```
+
+Here `cluster` is a name of a target cluster that is going to be cloned. If
+`timestamp` is not empty, then a new cluster will be cloned from an S3 bucket,
+that was created by operator,
+using the latest backup before the `timestamp`. In this case `uid` field is
+also mandatory - operator will use it to find an S3 bucket. You can find this
```

Comment:
the correct folder inside the S3 bucket.

Reply:
correct key maybe ? afaik there are no folders within S3 buckets, keys just mimic directory structure using names with slashes.

Reply:
although  directory (or folder) is universally understood in the context of S3 (I think WAL-E mentions it at least once on their page), using key would things less ambiguous, so +1